bundles/org.openhab.binding.opengarage/src/main/java/org/openhab/binding/opengarage/internal/OpenGarageHandler.java

```diff
@@ -49,8 +48,8 @@
 
     private long refreshInterval;
 
-    private @Nullable OpenGarageWebTargets webTargets;
-    private @Nullable ScheduledFuture<?> pollFuture;
+    private @NonNullByDefault({}) OpenGarageWebTargets webTargets;
+    private @NonNullByDefault({}) ScheduledFuture<?> pollFuture;
```

Comment:
That's not the correct solution. There are situations where `pollFuture` is indeed `null`.

Reply:
Apologies.. fixed in a1f3facd2f